test_utils.py

```diff
@@ -0,0 +1,8 @@
+import utils
+import yaml
+
+
+def get_test_list_hosts(cluster_id):
+    with open('/data/installer_dir/test_hosts_list.yaml', 'r') as file:
+        hosts_list = yaml.full_load(file)
+        return [utils.InventoryHost(host) for host in hosts_list if host['status'] != 'disabled']
```

Comment:
What are you testing here?
Where are the assertions, expectations?

Reply:
This repository does not has test files. This mode is meant to allow running this job as a container on laptop, and to check that ignitions are created and the relevant annotations are added ( instead of trying to run in on minikube.